packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/emitting/NotificationsEmitter.kt

```diff
@@ -41,6 +41,11 @@ open class NotificationsEmitter : Module(), NotificationListener {
     AsyncFunction<Bundle?>("getLastNotificationResponseAsync") {
       lastNotificationResponseBundle
     }
+
+    AsyncFunction("clearLastNotificationResponseAsync") {
+      lastNotificationResponseBundle = null
```

Comment:
If I'm not mistaken, this clears `lastNotificationResponseBundle` but the `useLastNotificationResponse` hook which is mentioned in the original issue https://github.com/expo/expo/issues/18712 will still have the old state?

Reply:
I'll have a look to see if I can reproduce this and come up with a clean fix.

Reply:
Fixed!